features/step_definitions/within_steps.rb

```diff
@@ -60,7 +60,8 @@
   'in the twitter govbox' => '#twitter_govbox',
   'in the site header' => '.l-site-header',
   'in the active site main navigation' => '.l-site-nav.main .active',
-  'in the active site sub navigation' => '.l-site-nav.sub .active'
+  'in the active site sub navigation' => '.l-site-nav.sub .active',
+  'in the SearchgovDomain URLs table' => 'tbody.messages #as_admin__searchgov_urls-messages'
```

Comment:
@ethanewing , that selector appears to be for just the section that holds the flash messages for that table. As we may want to use the `in the SearchgovDomain URLs table` selector more generically, I'd try the `as_admin__searchgov_urls-active-scaffold` div.